src/main/java/home/intexsoft/bank_application/BankAppRunner.java

```diff
@@ -0,0 +1,157 @@
+
+        MenuItem operationsItem = new MenuItem();
+        operationsItem.setName(Command.CommandType.OPERATIONS);
+        operationsItem.setParent(mainMenuItem);
+
+        MenuItem addMoneyTransferItem = new MenuItem();
+        addMoneyTransferItem.setCommand(Boolean.TRUE);
+        addMoneyTransferItem.setName(Command.CommandType.ADD_MONEY_TRANSFER);
+        addMoneyTransferItem.setParent(operationsItem);
+
+        MenuItem addSalaryPaymentItem = new MenuItem();
+        addSalaryPaymentItem.setCommand(Boolean.TRUE);
+        addSalaryPaymentItem.setName(Command.CommandType.ADD_SALARY_PAYMENT);
+        addSalaryPaymentItem.setParent(operationsItem);
+
+        MenuItem addCurrencyItem = new MenuItem();
+        addCurrencyItem.setCommand(Boolean.TRUE);
+        addCurrencyItem.setName(Command.CommandType.ADD_CURRENCY);
+        addCurrencyItem.setParent(banksItem);
+
+        MenuItem findTransactionsOfClientItem = new MenuItem();
+        findTransactionsOfClientItem.setCommand(Boolean.TRUE);
+        findTransactionsOfClientItem.setName(Command.CommandType.FIND_OPERATIONS_OF_CLIENT);
+        findTransactionsOfClientItem.setParent(banksItem);
+
+        MenuItem addBankAccountItem = new MenuItem();
+        addBankAccountItem.setCommand(Boolean.TRUE);
+        addBankAccountItem.setName(Command.CommandType.ADD_BANK_ACCOUNT);
+        addBankAccountItem.setParent(clientsItem);
+
+        mainMenuItem.getChildren().put("1", banksItem);
+        mainMenuItem.getChildren().put("2", clientsItem);
+        mainMenuItem.getChildren().put("3", operationsItem);
+
+        banksItem.getChildren().put("1", addBankItem);
+        banksItem.getChildren().put("2", deleteBankItem);
+        banksItem.getChildren().put("3", findClientsOfBankItem);
+        banksItem.getChildren().put("4", viewAllBanksItem);
+        banksItem.getChildren().put("5", addCurrencyItem);
+
+        clientsItem.getChildren().put("1", addClientItem);
+        clientsItem.getChildren().put("2", deleteClientItem);
+        clientsItem.getChildren().put("3", deleteAllClientsItem);
+        clientsItem.getChildren().put("4", findBankAccountOfClientItem);
+        clientsItem.getChildren().put("5", addBankAccountItem);
+
+        operationsItem.getChildren().put("1", addMoneyTransferItem);
+        operationsItem.getChildren().put("2", findTransactionsOfClientItem);
+        operationsItem.getChildren().put("3", addSalaryPaymentItem);
+
+        menu.setActiveItem(mainMenuItem);
+    }
+
+    private void runMenu() {
+        Log.debug("Starting of RunMenu");
+        System.out.println("Choose operation number to proceed");
+        System.out.println("Enter 'quit' to exit, enter 'back' to return to previous menu");
+        String userInput = "";
+        while (!"quit".equalsIgnoreCase(userInput)) {
+            menu.getActiveItem().getChildren().entrySet().forEach((child -> System.out.println(
+                    child.getKey() + " " + child.getValue().getName())));
```

Comment:
@DmitryZagorski, do not use explicit strings concatenation, recall String.format.